rpcs3/main.cpp

```diff
@@ -470,11 +492,33 @@ int main(int argc, char** argv)
 	}
 #endif
 
+	// If we're in headless, or non-gui, or we pass in the command line to exit after running, then we exit the
+	// application when the Emulator terminates (potentially returning success / error return code)
+	bool exit_after = parser.isSet(arg_headless) || parser.isSet(arg_no_gui) || parser.isSet(arg_exit_after);
+
+	if (parser.isSet(arg_install_pkgs) && !parser.value(arg_install_pkgs).isEmpty())
+	{
+		const auto pkg_paths = parser.value(arg_install_pkgs).split(',');
+		for (const auto pkg_path : pkg_paths)
+		{
+			if (!pkg_path.isEmpty())
+			{
+				if (!Emu.InstallPkg(pkg_path.toStdString()))
+				{
+					std::cerr << QCoreApplication::translate("Main", "Failed to InstallPkg \"%1\"\n").arg(pkg_path).toStdString();
+
+					if(exit_after)
```

Comment:
what's with the formatting again?
can you please start using our coding style guidelines?
It's really getting annoying

Reply:
Which particular part of the coding style guidelines are you referencing here?
Can you link to it?  The only one I've seen contains the wording "They aren't very strict rules since we want to be flexible..."
https://github.com/RPCS3/rpcs3/wiki/Coding-Style
And don't mention anything about whitespace, or braces/bracket position.

Reply:
common sense would show you that all other code doesn't look like this